lms/static/js/bookmarks/views/bookmarks_list.js

```diff
@@ -0,0 +1,124 @@
+;(function (define, undefined) {
+    'use strict';
+    define(['gettext', 'jquery', 'underscore', 'backbone', 'moment'],
+        function (gettext, $, _, Backbone, _moment) {
+
+        var moment = _moment || window.moment;
+
+        return Backbone.View.extend({
+
+            el: '#courseware-results-list',
+            coursewareContentElement: '#course-content',
+
+            errorIcon: '<i class="fa fa-fw fa-exclamation-triangle message-error" aria-hidden="true"></i>',
+            loadingIcon: '<i class="fa fa-fw fa-spinner fa-pulse message-in-progress" aria-hidden="true"></i>',
+
+            errorMessage: gettext('An error has occurred. Please try again.'),
+            loadingMessage: gettext('Loading'),
```

Comment:
can we also rename this to `isLoadingMessage` or `inProgressMessage`. Please also change all occurrences of simple `loading***` accordingly.